docs/usage-and-performance-tools/usage-analytics.md

```diff
@@ -19,7 +19,8 @@ These resources are useful for:
 Some things to keep in mind with this special Usage analytics collection:
 
 - Usage analytics is a view-only collection. Even admins can't curate it. It is eternal.
-- By default, only admins can view the Usage analytics collection (though admins can grant other groups view access to it). If you're upgrading from a version older than 48, people in groups with [monitoring access](../usage-and-performance-tools/usage-analytics.md) will also get access to the Usage analytics collection. But after that initial grandfathering in, the monitoring access privilege is unrelated to the Usage analytics collection; you'll need to specifically grant groups access to the Usage analytics collection.
+- By default, only admins can view the Usage analytics collection (though admins can grant other groups view access to it). If you're upgrading from a version older than 48, people in groups with monitoring access will also get access to the Usage analytics collection. But after that initial grandfathering in, the monitoring access privilege is unrelated to the Usage analytics collection; you'll need to specifically grant groups access to the Usage analytics collection.
+- We don't collect [Activity](#activity-log-model) and [View](#view-log-model) data on Metabase Open Source, so if you upgrade from Open Source to a Pro or Enterprise plan, you will not see usage data from before your upgrade in Usage Analytics.
```

Comment:
```suggestion
- Metabase Open Source edition (which the [Cloud Starter plan](https://www.metabase.com/pricing/) uses) doesn't collect [Activity](#activity-log-model) and [View](#view-log-model) data. If you upgrade from Open Source to a Pro or Enterprise plan, you'll only see data in Usage Analytics starting from when you upgraded.
```

- Adds reference to cloud plan
- Rephrases to make it Metabase the software, not "we".

Reply:
oh great call!